README.md

```diff
@@ -1,34 +1,14 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Kowalski I Spółka
 
-## Getting Started
 
-First, run the development server:
+<h3 align="center">Home</h3>
 
-```bash
```

Comment:
Avoid removing instructions on how to develop. It's cool that you made a README but leave project start info